docs/docs-content/tutorials/tutorials.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_label: "Palette Tutorials"
+title: "Welcome to Palette Tutorials!"
+sidebar_position: 0
+pagination_next: null
+---
+
+The tutorials section provides you with hands-on tutorials using Spectro Cloud Palette. It covers everything you need to
+begin leveraging its Kubernetes full-stack management at scale. It covers the following key components.
+
+### Profiles
```

Comment:
We can keep this section for now, but what will we do when we have all the topics? Having a paragraph for each section may be a bit too much. I honestly think we can do without the sections below and just dive into the cards. We could use the section below to highlight new tutorials, specific topics, or important series, such as the getting started. I'll defer to you both on what approach you prefer.

Reply:
We wanted to give a little bit of context for now. When we have more sections, we could also consider making landing pages for each section. We felt it was a bit sparse to have only the introductions and the cards on the page.